grammar.js

```diff
@@ -4,19 +4,18 @@ const PHP = require('tree-sitter-php/grammar');
 // TODO array return types https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html#arrays
 // PHPDoc tags: https://docs.phpdoc.org/3.0/guide/references/phpdoc/tags/index.html#tag-reference
 // PHPDoc docs appear to use these conventions
+// TODO inline tags in inline tags (allowed for @internal)
 //   @verbatimElement [required element] [<optional element>]
 
 module.exports = grammar({
   name: 'phpdoc',
 
   extras: $ => [
-    token(
-      seq(/(\r)|(\r\n)|(\n)/, /[ \t]*/, repeat('*'), /[ \t]*/)
-    ),
-    token(
-      seq(/[ \t]*/, repeat('*'), /[ \t]*/)
```

Comment:
This IMO would filter out way to much, e.g. any '*' inside a description. The first extra above should already be sufficient to deal with line starts. But maybe I miss something here.

Reply:
Same here. If it's not needed: ✂️